### ISSUE TITLE
Typescript types missing -> can't use in typescript ?

### ISSUE THREAD

**Issue by andreimc** [fannypackui/fannypack#85]
Just wondering I see this is wrriten in typescript but I install it and:

```Could not find a declaration file for module 'fannypack'. 'blah/node_modules/fannypack/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/fannypack` if it exists or add a new declaration (.d.ts) file containing `declare module 'fannypack';`ts(7016)```

**Comment by jxom** [comment]
Looks like the ts/ folder disappeared in the latest release. I just published `4.20.13` which should have the types back in!

**Comment by patrick91** [comment]
This seems to have happened again in latest release `4.21.21`

**Comment by jxom** [comment]
~Looks like the TS compiler broke itself as I was publishing, let me republish...~

Fixed on `4.21.22`
